docs/scos/dev/guidelines/testing-guidelines/executing-tests/test-the-glue-backend-api.md

```diff
@@ -0,0 +1,271 @@
+     */
+    public function buildFixtures(ModuleBackendApiTester $I): FixturesContainerInterface
+    {
+        $this->createUser($I);
+
+        return $this;
+    }
+
+    /**
+     * @param \PyzTest\Glue\ModuleBackend\ModuleBackendApiTester $I
+     *
+     * @return void
+     */
+    protected function createUser(ModuleBackendApiTester $I): void
+    {
+        $this->userTransfer = $I->haveUser([
+            UserTransfer::PASSWORD => static::TEST_USER_PASSWORD,
+            UserTransfer::USERNAME => static::TEST_USER_NAME,
+        ]);
+        
+        // Override encrypted password with plain password for further testing purposes.
+        $this->userTransfer->setPassword(static::TEST_USER_PASSWORD);
+    }
+}
+```
+
+2. Run the following command to generate fixtures:
+
+```bash
+docker/sdk testing codecept fixtures
+```
+
+#### 4. Create the test cest file
+
+**tests/PyzTest/Glue/ModuleBackend/JsonApi/ModuleBackendJsonApiCest.php**
+
+```php
+<?php
+
+namespace PyzTest\Glue\ModuleBackend\JsonApi;
+
+use Codeception\Util\HttpCode;
+use Pyz\Glue\ModuleRestApi\ModuleRestApiConfig;
+use PyzTest\Glue\ModuleBackend\ModuleBackendApiTester;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group PyzTest
+ * @group Glue
+ * @group ModuleBackend
+ * @group JsonApi
+ * @group ModuleBackendJsonApiCest
+ * Add your own group annotations below this line
+ * @group EndToEnd
+ */
+class ModuleBackendJsonApiCest
+{
+    /**
+     * @var \PyzTest\Glue\ModuleBackend\RestApi\ModuleBackendJsonApiFixtures
+     */
+    protected ModuleBackendJsonApiFixtures $fixtures;
+
+    /**
+     * @param \PyzTest\Glue\ModuleBackend\ModuleBackendApiTester $I
+     *
+     * @return void
+     */
+    public function loadFixtures(ModuleBackendApiTester $I): void
+    {
+        /** @var \PyzTest\Glue\ModuleBackend\JsonApi\ModuleBackendJsonApiFixtures $fixtures */
+        $fixtures = $I->loadFixtures(ModuleBackendJsonApiFixtures::class);
+        $this->fixtures = $fixtures;
+    }
+
+    /**
+     * @depends loadFixtures
+     *
+     * @param \PyzTest\Glue\ModuleBackend\ModuleBackendApiTester $I
+     *
+     * @return void
+     */
+    public function requestGetModule(ModuleBackendApiTester $I): void
+    {
+        // Arrange
+        $oauthResponseTransfer = $I->haveAuthorizationToBackendAPI($this->fixtures->getUserTransfer());
+        $I->amBearerAuthenticated($oauthResponseTransfer->getAccessToken());
+        
+        // Act
+        $I->sendGET(
+            $I->formatUrl(ModuleRestApiConfig::RESOURCE_MODULE),
+        );
+
+        // Assert
+        $I->seeResponseCodeIs(HttpCode::OK);
+        $I->seeResponseIsJson();
+        $I->seeResponseMatchesOpenApiSchema();
+    }
+}
+```
+
+#### 5. Run the test
+
+Run the following command to execute the test:
+
+```bash
+docker/sdk testing codecept run -c tests/PyzTest/Glue/ModuleBackend
+```
+
+Once the testing process is complete, you get the result of each individual test.
```

Comment:
* The result of each individual test will be displayed once the testing process is complete.